packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.scss

```diff
@@ -90,6 +90,7 @@
   @include type.type-style('body-compact-02');
   color: $text-02;
   margin-top: 0.375rem;
+  text-align: left;
```

Comment:
The demographics and the identifiers were bought center aligned (see screen shots) this and the line below just left-aligns them.